src/test/java/uk/gov/hmcts/cmc/claimstore/services/interest/InterestCalculationServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.services.interest;
+
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Clock;
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class InterestCalculationServiceTest {
+
+    private static final LocalDate START_DATE = LocalDate.now();
+    private static final BigDecimal NEGATIVE = BigDecimal.valueOf(-12);
+
+    private InterestCalculationService service = new InterestCalculationService(Clock.systemDefaultZone());
+
+    @Test(expected = IllegalArgumentException.class)
+    public void shouldIllegalArgumentExceptionWhenAmountIsNegative() {
```

Comment:
Please correct to be `shouldThrow...`. A `shouldIllegal...` does not read well.

Reply:
changed